src/Service/BattleEngine/BattlePhase.php

```diff
@@ -10,6 +10,9 @@
 use FrankProjects\UltimateWarfare\Entity\WorldRegionUnit;
 use RuntimeException;
 
+/**
+ * @property array $attackerGameUnits
```

Comment:
Is there a specific reason you use @property to document variables instead of the docblocks I use?

Reply:
This project is using strict typing, so any editor or documentation tool should already see $attackerGameUnits is an array as it is defined and enforced as array, see line 24

Reply:
Got confused due to the wrong typehint in Battlephase.